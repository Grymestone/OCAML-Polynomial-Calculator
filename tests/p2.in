(2a + 4a)^2
