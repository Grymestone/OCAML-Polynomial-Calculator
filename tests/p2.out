(((2*a)+(4*a)))^2
36x^2